rasa/core/policies/memoization.py

```diff
@@ -135,7 +136,20 @@ def _add_states_to_lookup(
     def _create_feature_key(self, states: List[Dict]) -> Text:
         from rasa.utils import io
 
-        feature_str = json.dumps(states, sort_keys=True).replace('"', "")
+        new_states = []
```

Comment:
memoization is broken, anyhow?

Reply:
it is! I thought for a second there was a quick/simple fix... which in reality there wasn't